release-content/0.16/release-notes/virtual_geometry_improvements.md

```diff
@@ -0,0 +1,7 @@
+Virtual geometry (the `meshlet` feature) got some performance improvements in Bevy 0.16.
```

Comment:
I think we should add a few words that explain what virtual geometry is for folks who don't know already.